airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/gradle.py

```diff
@@ -32,210 +36,129 @@ class GradleTask(Step, ABC):
 
     context: ConnectorContext
 
-    GRADLE_DEP_CACHE_PATH = "/root/gradle-cache"
-    GRADLE_HOME_PATH = "/root/.gradle"
-    STATIC_GRADLE_OPTIONS = ("--no-daemon", "--no-watch-fs", "--build-cache", "--scan", "--console=plain")
-    CDK_MAVEN_METADATA_URL = (
-        "https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/io/airbyte/cdk/airbyte-cdk-core/maven-metadata.xml"
-    )
+    STATIC_GRADLE_OPTIONS = ("--build-cache", "--scan")
```

Comment:
Which options shall we keep compared to the containerized implementation?

Reply:
we should keep --no-daemon and --no-watch-fs since this is a single build so we don't need to incur the overhead of start the daemon/watching the fs that is mostly useful for systems that are long running.

Reply:
I don't know about --no-daemon, aren't we going to call gradle multiple times in the same VM now?

Reply:
good question - I'm assuming we only run 1 connector test per CI runner and most devs will not invoke `airbyte-ci` locally, and will instead directly run gradle. Augustine, does that sounds right?

Reply:
> I'm assuming we only run 1 connector test per CI runner
Nope, the default airbyte-ci implementation is to run all modified connectors tests in the same runner, concurrently for python connectors, sequentially for java connectors as concurrent access to the gradle cache is not possible. I can try to revisit this logic later. Colocating execution on the same runner has benefits for python connectors for dagger caching. Following this change colocation for java connectors makes less sense and hurts DX.

In any case we do run multiple gradle command in a single connector pipeline execution: `distTar`, `unitTests`, `integrationTests`. 

**TLDR;**: I don't think we should add `--no-daemon` back. I'll re-add `--no-watch-fs`. Thanks for your inputs.